css/index.css

```diff
@@ -143,4 +143,39 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+.navmain{
+    width: 60%;
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 2%;
+
+}
+.navmain a{
+    color:black;
+    text-decoration: none;
+}
+.topnav{
+    width:100%;
+    display:flex;
+    justify-content: center;
+
+}
+.jumbohome{
+    display:flex;
+    justify-content: center;
+    margin: 2rem;
+}
+.abouttop{
+    display:flex
+    justify-content: center;
+    border-bottom: dotted black;
+}
+.boxs{
+    border-bottom: dotted black;   
+}
+button{
+border-bottom: dotted black;
+
```

Comment:
It looks like you are making good progress on this.  The index.html page is almost done, the boxes can be colored as well per the Readme.